docs/walletkit/ios/experimental/chain-abstraction.mdx

```diff
@@ -0,0 +1,141 @@
+# Chain Abstraction
+
+Chain Abstraction in WalletKit enables users with stablecoins on any network to spend them on-the-fly on a different network. Our Chain Abstraction solution provides a toolkit for wallet developers to integrate this complex functionality using WalletKit.
+
+For example, when an app requests a 100 USDC payment on Base network but the user only has USDC on Arbitrum, WalletKit offers methods to detect this mismatch, generate necessary transactions, track the cross-chain transfer, and complete the original transaction after bridging finishes.
+
+## How It Works
+
+:::note
+Apps need to pass `gas` as null, while sending a transaction to allow proper gas estimation by the wallet. Refer to this [guide](../../../appkit/next/experimental/chain-abstraction.mdx) for more details.
+:::
+
+When a app requests a transaction through `eth_sendTransaction`, you need to:
```

Comment:
we discussed on the last call to not mention "app requests a transaction through `eth_sendTransaction`" but explain it directly from wallet perspective